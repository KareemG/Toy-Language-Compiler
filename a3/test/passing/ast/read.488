% Read statement

{
    var x : integer

    read x
}