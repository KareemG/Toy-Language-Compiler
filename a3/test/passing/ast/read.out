 { 
 var x : integer
 read x
 }
